DashWallet/Sources/Models/CrowdNode/CrowdNodeConstants.swift

```diff
@@ -15,21 +15,20 @@
 //  limitations under the License.
 //
 
-struct CrowdNodeConstants {
+enum CrowdNodeConstants {
```

Comment:
@Syn-McJ I think, that's because you only have static constants inside

Reply:
yeah, found this https://github.com/kodecocodes/swift-style-guide#constants 

> The advantage of using a case-less enumeration is that it can't accidentally be instantiated and works as a pure namespace.

Reply:
Makes sense